### DISCUSSION TITLE
Whether char pointer can be converted to a float pointer

### DISCUSSION THREAD

**Discussion by Lester-1** [nothings/stb#1417] [Q&A]
I found that an unsigned char pointer was obtained during image reading. I want to ask whether there is an implementation that can eventually be read as float. I tried to use the stbi_loadf method and found that the result is very different from that of the stbi_load method.

**Comment by nothings** [comment]
stbi_loadf returns the pixel values as normalized floats, i.e. from 0..1 instead of 0..255.
